stable/mlflow/templates/service.yaml

```diff
@@ -17,6 +17,12 @@ spec:
       port: {{ .Values.service.port }}
       protocol: TCP
       targetPort: {{ .Values.service.targetPort }}
+{{- if .Values.service.https.enable }}
+    - name: https
+      port: {{ .Values.service.https.port }}
+      protocol: TCP
+      targetPort: {{ .Values.service.targetPort }}
```

Comment:
Is this going to work if it is the same as the http targetPort?

Reply:
Based that off of this doc: https://aws.amazon.com/premiumsupport/knowledge-center/terminate-https-traffic-eks-acm/

I hope it works, but might be a good idea to let the targetPorts be different as well

Reply:
I'll try to find a way to test locally without merging this to master to short-circuit this cycle time 🤔

Reply:
Hmm, that link seems pretty convincing.  I was just making sure this was intentional and not a copy-paste oversight.

Reply:
> I'll try to find a way to test locally without merging this to master

If you are using helmfiles, you can simply change the source like this:

```yaml
- name: chart-release-name
  # chart: deliveryhero/mlflow
  chart: /path/to/your/local/chart/copy
  version: 7.6.21
```

Reply:
Thanks! Testing this way was quite useful. Since the deployment only has 1 possible port, I won't make the https block customizable